src/PrestaShopBundle/Controller/Admin/ModuleController.php

```diff
@@ -706,7 +766,29 @@ public function getModuleCartAction($moduleId)
             '@PrestaShop/Admin/Module/Includes/modal_read_more_content.html.twig',
             array(
                 'module' => $moduleToPresent,
+                'level' => $this->authorizationLevel(),
             )
         );
     }
+
+    /**
+     * Return the type of authorization on module page.
+     *
+     * @return int(integer)
+     */
+    public function authorizationLevel()
+    {
+        switch (true) {
```

Comment:
I'm pretty sure there is something more readable to do instead of a switch hack (but I like it :))